setup.py

```diff
@@ -49,7 +48,7 @@ def read(fname):
     test_requirements = fh.readlines()
 
 python_requires = '>=3.5.*,<4'
-setup(name='mbed-greentea',
+setup(name='greentea',
```

Comment:
Changing the name will mean we create an entirely new project on PyPI. Is that the intention? I suggest we keep the "mbed-greentea" project name as there is already a "greentea" project on PyPI, which has nothing to do with Arm or Mbed. So, this won't work when we try to release, as PyPI doesn't allow duplicated project names.

Reply:
Yes to create a new project for standalone but as you said there is already one `greentea` project in PyPI, so I will keep `mbed-greentea`